.github/CODEOWNERS

```diff
@@ -395,6 +395,36 @@ flang/**/CodeGen/ @llvm/pr-subscribers-flang-codegen
 /llvm/utils/TableGen/CodeGenDAG* @llvm/pr-subscribers-selectiondag
 /llvm/utils/TableGen/DAGISel* @llvm/pr-subscribers-selectiondag
 
+# register allocation
+/llvm/include/CodeGen/RegAlloc* @llvm/pr-subscribers-regalloc
+/llvm/include/CodeGen/*Regalloc* @llvm/pr-subscribers-regalloc
+/llvm/include/CodeGen/Register* @llvm/pr-subscribers-regalloc
+/llvm/include/CodeGen/LiveReg* @llvm/pr-subscribers-regalloc
+/llvm/include/CodeGen/LiveInterval* @llvm/pr-subscribers-regalloc
+/llvm/include/CodeGen/LiveVariables* @llvm/pr-subscribers-regalloc
+/llvm/include/CodeGen/LiveRange* @llvm/pr-subscribers-regalloc
+/llvm/include/CodeGen/SlotIndexes.h @llvm/pr-subscribers-regalloc
+/llvm/include/CodeGen/VirtRegMap.h @llvm/pr-subscribers-regalloc
+/llvm/include/PBQP/ @llvm/pr-subscribers-regalloc
+/llvm/include/PBQPRAConstraint.h @llvm/pr-subscribers-regalloc
+/llvm/include/llvm/CodeGen/Spiller.h @llvm/pr-subscribers-regalloc
+/llvm/lib/CodeGen/*RegAlloc* @llvm/pr-subscribers-regalloc
```

Comment:
Wouldn't `llvm/**/*RegAlloc` be enough to cover include/ and lib/?
(you don't need the trailing `*`, it seems that .gitignore format matches on prefix from what I gathered)

Reply:
I have no idea how to test if this is working or not

Reply:
This is what we use for many other patterns in this file :)

But the way I test it is by trying with `.gitignore` (GitHub claims that they use the same format).

So I modify a file (or create a new path), check that it shows up in `git status`, then I add the pattern to `.gitignore` and check that it does not show up in `git status` output anymore!